src/components/views/settings/ChangePassword.js

```diff
@@ -213,29 +347,39 @@ export default class ChangePassword extends React.Component {
                     <form className={this.props.className} onSubmit={this.onClickChange}>
                         <div className={rowClassName}>
                             <Field
+                                ref={field => this[FIELD_OLD_PASSWORD] = field}
                                 type="password"
                                 label={_t('Current password')}
                                 value={this.state.oldPassword}
                                 onChange={this.onChangeOldPassword}
+                                onValidate={this.onOldPasswordValidate}
+                                onBlur={() => CountlyAnalytics.instance.track("onboarding_registration_email_blur")}
```

Comment:
this shouldn't be here, this isn't onboarding and there isn't a matching focus track